db/migrate/20220111170303_fix_child_container_manager_sti.rb

```diff
@@ -46,18 +46,17 @@ def up
         ServiceParametersSet.in_my_region.where(:ext_management_system => managers).update_all(:type => "#{provider_klass}::ServiceParametersSet")
       end
     end
+  end
```

Comment:
oops!  I can't believe 6.1 didn't fail previously with this

Reply:
wow